django_inbound_email/backends/mailgun.py

```diff
@@ -43,9 +43,18 @@ def parse(self, request):
             )
             html = request.POST.get('stripped-html')
             from_email = request.POST.get('sender')
-            to_email = request.POST.get('recipient').split(',')
-            cc = request.POST.get('cc', '').split(',')
-            bcc = request.POST.get('bcc', '').split(',')
+            to_email = request.POST.get('recipient')
```

Comment:
This change makes things nicer, setting the params to `None` instead of `['']`, but it is a slight change in interface. @hugorodgerbrown what do you think?